client/js/helpers/parse.ts

```diff
@@ -212,6 +215,25 @@ function parse(text: string, message?: ClientMessage, network?: ClientNetwork) {
 					default: () => fragments,
 				}
 			);
+		} else if (textPart.seb) {
+			const baseUrl = "https://d2nj10p1euioo5.cloudfront.net/";
+
+			const name = textPart.seb[1];
+			const filename: string = textPart.seb[0];
+			const metadata = filename.match(/[a-z0-9]+_?(\d+)?/i);
+			const height = metadata[1] !== "undefined" ? metadata[1] : "auto";
```

Comment:
I'm not sure if the array returned by `match` coerces the literal value of `undefined` to be a string (i.e. `"undefined"`), but if that's not the case, this conditional will *always* return `metadata[1]` unless we had a seb filename with a pattern like `<name>_undefined.<ext>`. 

I think we have two options here:

```ts
const height = metadata[1] !== undefined ? metadata[1] : "auto";
```

or

```ts
const height = metadata[1] || "auto";
```